### PR TITLE
Update query-syntax.md

### PR DESCRIPTION
**Pull request by heyitswei**
Word polish (wIp), until :::info

### CODE REVIEW

File: docs/query-syntax.md [hunk 1]
```diff
@@ -45,6 +47,8 @@ WHERE cpu_usage >= 99
 The above example continuously evaluates the filter expression on the new events in the stream `device_utils` to filter out events
 which have `cpu_usage` less than 99. The final events will be streamed to clients.
 
+\
```
Comment (jovezhong) [inline docs/query-syntax.md:50]: what's this

File: docs/query-syntax.md [hunk 2]
```diff
@@ -73,6 +77,8 @@ Like in [Streaming Tail](#streaming-tailing), Timeplus continuously monitors new
 continuously does **incremental** count aggregation. Whenever the specified delay interval is up, project the current aggregation result
 to clients.
 
+\
```
Comment (jovezhong) [inline docs/query-syntax.md:80]: same issue

File: docs/query-syntax.md [hunk 3]
```diff
@@ -158,6 +164,8 @@ GROUP BY device, window_end
 EMIT AFTER WATERMARK DELAY 2s;
 ```
 
+\
```
Comment (jovezhong) [inline docs/query-syntax.md:167]: same issue

**Review by jovezhong — CHANGES_REQUESTED** [review]
some unexpected \

File: docs/query-syntax.md (hunk 1 above)
Comment (heyitswei) [reply docs/query-syntax.md:50]: Hi @jovezhong, I was trying to put in an extra line break, to visually separately the sections better. Referred to this Github section in this [tutorial ](https://www.w3schools.io/file/markdown-line-break/)-- is there a better way to add the line break?

File: docs/query-syntax.md (hunk 2 above)
Comment (jovezhong) [reply docs/query-syntax.md:80]: In general, we should not care too much about line breaks in documentation. The martin/etc doesn't matter in documentations. For me, if I really have to add new lines, I will use <br>

File: docs/query-syntax.md [hunk 4]
```diff
@@ -13,18 +13,18 @@ EMIT <window_emit_policy>
...
+2. `seek_to=<timestamp|earliest|latest>`. A setting which tells Timeplus to seek old data in the streaming storage by timestamp. It can be a relative timestamp or an absolute timestamp. By default, it is `latest`, which tells Timeplus to not seek old data. Example:`seek_to='2022-01-12 06:00:00.000'`, ` seek_to='-2h'`, or ` seek_to='earliest'` 
 
 :::info
 
-Please note, since Jan 2023, `SETTINGS seek_to=..` is no longer recommended to use. Please use `WHERE _tp_time>='2023-01-01'` or something similar. `_tp_time` is the special timestamp column in each raw stream to represent the event time. You can use `>`, `<`, `BETWEEN .. AND` operations to filter the data in Timeplus stream storage.
+Please note, as of Jan 2023, we no longer recommend you use `SETTINGS seek_to=..`. Please use `WHERE _tp_time>='2023-01-01'` or similar. `_tp_time` is the special timestamp column in each raw stream to represent the event time. You can use `>`, `<`, `BETWEEN .. AND` operations to filter the data in Timeplus stream storage.
 
-:::
+:::
```
Comment (jovezhong) [inline docs/query-syntax.md]: the whitespace after ::: is very minor, but you don't add them either

**Review by jovezhong — APPROVED** [review]
basically okay, pls merge by yourself

**Pull request merged**